### PR TITLE
BZ2030453: Add initial steps to add server certificate

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.6+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2030453

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/add-server-certs/security/certificates/api-server.html#customize-certificates-api-add-named_api-server-certificates

Additional information:

@xiaojiey

### CODE REVIEW

**Comment by xiaojiey** [comment]
@wangke19 not sure if you are the right person or not. Could you please take a look? Thanks.

**Comment by gangwgr** [comment]
As Checked doc providing lgtm in PR 
/lgtm

File: modules/customize-certificates-api-add-named.adoc [hunk 1]
```diff
@@ -31,6 +31,20 @@ cluster in a degraded state.
 
 .Procedure
 
+. Login to the new API with the `kubeadmin` user.
```
Comment (opayne1) [inline modules/customize-certificates-api-add-named.adoc:34]: Should this be 

```suggestion
. Login to the new API as the `kubeadmin` user.
```
or 

```suggestion
. Login to the new API with `kubeadmin` user privileges.
```

**Review by opayne1 — COMMENTED** [review]
Just one question, other than that it lgtm.

File: modules/customize-certificates-api-add-named.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/customize-certificates-api-add-named.adoc]: It should be . "Login to the new API as the `kubeadmin` user."

Comment (xenolinux) [reply modules/customize-certificates-api-add-named.adoc]: Addressed

**Pull request merged**